Program.cs

```diff
@@ -1,10 +1,13 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿//An isogram is a word that has no duplicate letters.
+//Create a function that takes a string and returns either true or false depending on whether it's an "isogram". 
+
+Console.WriteLine ("Enter a string: ");

```

Comment:
Handle this error.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/aab1fab5-81c3-4fa4-bd4d-a28666bb0567)